.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,63 @@
+name: "Coding Standards"
+
+on:
+  pull_request:
+    branches:
+      - "[0-9]+.[0-9]+"
+      - "feature/*"
+    paths-ignore:
+      - "docs/**"
+  push:
+    branches:
+      - "[0-9]+.[0-9]+"
+      - "feature/*"
+    paths-ignore:
+      - "docs/**"
+
+env:
+  PHP_VERSION: "8.2"
+  DRIVER_VERSION: "mongodb/mongo-php-driver@master"
```

Comment:
Does this actually require 1.17-dev, or was this just copied from PHPLIB? I would think we can rely on the stable release here.